helm/oms-agent/values.yaml

```diff
@@ -0,0 +1,133 @@
+# TEMPLATE-APP: This is set as a reasonable default, feel free to change.
+
+name: oms-agent
+namespace: oms-agent
+
+omsagent:
+  image:
+    repository: quay.io/giantswarm/ciprod
+    tag: ciprod06112021
```

Comment:
app name is `oms-agent` but container image name is `ciprod`?

Reply:
yes that is the MS naming crap that i wanted to have aligned at least unfortunately

Reply:
ok :+1: